gazelle/internal/swift/bzlmod_test.go

```diff
@@ -47,3 +47,44 @@ use_repo(
 `
 	assert.Equal(t, expected, actual)
 }
+
+func TestBzlmodStanzasWithCustomDepsIndex(t *testing.T) {
+	awesomeRepoId := "awesome-repo"
+	awesomePkg := &swift.Package{
+		Name:     swift.RepoNameFromIdentity(awesomeRepoId),
+		Identity: awesomeRepoId,
+		Remote: &swift.RemotePackage{
+			Commit: "12345",
+			Remote: "https://github.com/example/awesome-repo",
+		},
+	}
+	anotherRepoID := "another-repo"
+	anotherPkg := &swift.Package{
+		Name:     swift.RepoNameFromIdentity(anotherRepoID),
+		Identity: anotherRepoID,
+		Local: &swift.LocalPackage{
+			Path: "path/to/another",
+		},
+	}
+
+	di := swift.NewDependencyIndex()
+	di.AddPackage(awesomePkg, anotherPkg)
+	di.AddDirectDependency(awesomeRepoId, anotherRepoID)
+
+	actual, err := swift.BzlmodStanzas(di, "path/to/root", "path/to/root/swift", "path/to/root/swift/swift_deps_index.json")
```

Comment:
Can this test not use the default name `swift_deps_index.json` (e.g., `deps_index.json`)?

Reply:
Updated!